src/dmd/backend/cgsched.d

```diff
@@ -188,485 +234,497 @@ enum
-        EA,     F,              // CMP
-
+      [ EA,     F|EA ],           // ADD
+      [ EA,     F|EA ],           // OR
+      [ F|EA,   F|EA ],           // ADC
+      [ F|EA,   F|EA ],           // SBB
+      [ EA,     F|EA ],           // AND
+      [ EA,     F|EA ],           // SUB
+      [ EA,     F|EA ],           // XOR
+      [ EA,     F    ],           // CMP
+    ],
+    [
         // Grp 3
-        EA,     F,              // TEST EA,imm
-        N,      N,              // reserved
-        EA,     EA,             // NOT
-        EA,     F|EA,           // NEG
-        mAX|EA, F|mAX|mDX,      // MUL
-        mAX|EA, F|mAX|mDX,      // IMUL
-        mAX|mDX|EA,     F|mAX|mDX,      // DIV
-#if 0
+      [ EA,     F ],              // TEST EA,imm
+      [ N,      N ],              // reserved
+      [ EA,     EA ],             // NOT
+      [ EA,     F|EA ],           // NEG
+      [ mAX|EA, F|mAX|mDX ],      // MUL
+      [ mAX|EA, F|mAX|mDX ],      // IMUL
+      [ mAX|mDX|EA, F|mAX|mDX ],  // DIV
+
         // Could generate an exception we want to catch
-        mAX|mDX|EA|N,   F|mAX|mDX|N,    // IDIV
-#else
-        mAX|mDX|EA,     F|mAX|mDX,      // IDIV
-#endif
+        //mAX|mDX|EA|N,   F|mAX|mDX|N,    // IDIV
 
+      [ mAX|mDX|EA,     F|mAX|mDX ],      // IDIV
+    ],
+    [
         // Grp 5
-        EA,     F|EA,           // INC Ev
-        EA,     F|EA,           // DEC Ev
-        N|EA,   N,              // CALL Ev
-        N|EA,   N,              // CALL eP
-        N|EA,   N,              // JMP Ev
-        N|EA,   N,              // JMP Ep
-        mSP|EA, mSP|mMEM,       // PUSH Ev
-        N,      N,              // reserved
-
+      [ EA,     F|EA ],           // INC Ev
+      [ EA,     F|EA ],           // DEC Ev
+      [ N|EA,   N ],              // CALL Ev
+      [ N|EA,   N ],              // CALL eP
+      [ N|EA,   N ],              // JMP Ev
+      [ N|EA,   N ],              // JMP Ep
+      [ mSP|EA, mSP|mMEM ],       // PUSH Ev
+      [ N,      N ],              // reserved
+    ],
+    [
         // Grp 3, byte version
-        EA|B,   F,              // TEST EA,imm
-        N,      N,              // reserved
-        EA|B,   EA|B,           // NOT
-        EA|B,   F|EA|B,         // NEG
-        mAX|EA, F|mAX,          // MUL
-        mAX|EA, F|mAX,          // IMUL
-        mAX|EA, F|mAX,          // DIV
-#if 0
+      [ EA|B,   F ],              // TEST EA,imm
+      [ N,      N ],              // reserved
+      [ EA|B,   EA|B ],           // NOT
+      [ EA|B,   F|EA|B ],         // NEG
+      [ mAX|EA, F|mAX ],          // MUL
+      [ mAX|EA, F|mAX ],          // IMUL
+      [ mAX|EA, F|mAX ],          // DIV
+
         // Could generate an exception we want to catch
-        mAX|EA|N,       F|mAX|N,        // IDIV
-#else
-        mAX|EA, F|mAX,          // IDIV
-#endif
+        //mAX|EA|N,       F|mAX|N,        // IDIV
 
-};
+      [ mAX|EA, F|mAX ],          // IDIV
+    ]
```

Comment:
It only uses four top-level elements, not 8.

Reply:
So does the original!